README.md

```diff
@@ -70,6 +70,62 @@ As an alternative to the web component, a Sass mixin can be used to apply button
 
 The `primary` and `disabled` attributes work the same way as they do with the web component.
 
```

Comment:
Once this and the button-icon stuff goes in, let's take another look at the screenshots we're including here and see if we can come up with something better -- similar to what's on design.d2l maybe.